Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1180,21 +1180,14 @@ private void ReconcilePointers()
                 }
                 if (gazePointer != null)
                 {
-                    bool wasGazePointerActive = gazePointerStateMachine.IsGazePointerActive;
-
                     gazePointerStateMachine.UpdateState(
                         NumNearPointersActive,
                         NumFarPointersActive,
                         numFarPointersWithoutCursorActive,
                         CoreServices.InputSystem.EyeGazeProvider.IsEyeTrackingEnabledAndValid);
 
                     bool isGazePointerActive = gazePointerStateMachine.IsGazePointerActive;
-
-                    if (wasGazePointerActive != isGazePointerActive)
-                    {
-                        // The gaze cursor's visibility is controlled by IsInteractionEnabled
-                        gazePointer.IsInteractionEnabled = isGazePointerActive;
-                    }
+                    gazePointer.IsInteractionEnabled = isGazePointerActive;
```

Comment:
Is this change functionally doing anything? Looking at the code it looks like IsInteractionEnabled already has diff detection anyway so it looks like this change is just cleanup/goodness.

Reply:
There was an edge case we could get where when the application was paused and then resumed, the gaze cursor would stay active on the screen even when new hands were brought up (this is because we now disable and then enable all services on application pause)